### PR TITLE
dnf-plugins-core/download: add tests for --debugsource

### PR DESCRIPTION
**Pull request by lukash**
Adds tests for the new --debugsource argument of the download plugin.

https://bugzilla.redhat.com/show_bug.cgi?id=1637008

### CODE REVIEW

**Comment by lukash** [comment]
`dnf-plugins-core` PR: https://github.com/rpm-software-management/dnf-plugins-core/pull/353

**Comment by lukash** [comment]
I've rebased this on the latest master changes.

File: dnf-behave-tests/features/plugins-core/download-debuginfo.feature [hunk 1]
```diff
@@ -46,3 +46,67 @@ Scenario: Download an existing --debuginfo RPM with --verbose option
...
+
+@not.with_os=rhel__eq__8
+@bz1637008
+Scenario: Download source, debuginfo and debugsource for an existing RPM
+  Given I use repository "dnf-ci-fedora-updates" as http
+   When I execute dnf with args "download --source --debuginfo --debugsource libzstd"
+   Then the exit code is 0
+    And stdout contains "zstd-1.3.6-1.fc29.src.rpm"
+    # the file name below is cut off in the output table
+    And stdout contains "libzstd-debuginfo-1.3.6-1.fc29.x86_64.rp"
```
Comment (pkratoch) [inline dnf-behave-tests/features/plugins-core/download-debuginfo.feature]: Oh, just one more thing - there is a little typo:  "rp" instead of "rpm" :)

Comment (lukash) [reply dnf-behave-tests/features/plugins-core/download-debuginfo.feature]: Oh, no, that is not a typo, sadly. The name is too long and gets cut off in the output table (as the comment says :slightly_smiling_face: )

Comment (pkratoch) [reply dnf-behave-tests/features/plugins-core/download-debuginfo.feature]: Sorry, I don't get how I could have ignored the comment while reading.

**Pull request merged**